source/installation/install-on-an-unattended-server/debian-server-installation/debian-server-installation.txt

```diff
@@ -27,5 +27,5 @@ Get Started
    :hidden:
    :titlesonly:
 
-   /installation/install-on-an-unattended-server/debian-server-installation/install-debian-server
-   /installation/install-on-an-unattended-server/debian-server-installation/debian-system-service
+   Install </installation/install-on-an-unattended-server/debian-server-installation/install-debian-server>
+   Run as a System Service </installation/install-on-an-unattended-server/debian-server-installation/debian-system-service>
```

Comment:
[Suggestion] Drop the indefinite article for succinctness: "Run as System Service".

Reply:
Done